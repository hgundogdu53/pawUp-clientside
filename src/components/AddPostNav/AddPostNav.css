.nav-title {
    display:flex;
    flex-direction: row;
    justify-content: flex-start;
    padding-top: 0;
}

@media only screen and (min-width: 320px) and (max-width: 760px) {
    
    .bm-burger-button {
      position: fixed;
      width: 28px;
      height: 20px;
      right: 28px;
      top: 28px;
      color: #bdc3c7;
    }
    .bm-cross-button {
      height: 18px;
      width: 18px;
    }
    .icon {
      width: 3rem;
      height: 3rem;
    }
    .nav-title .title {
      text-align: center;
      margin-top:6%;
      padding-left: 6%;
      font-size: 36px;    
    }
  
  
  }