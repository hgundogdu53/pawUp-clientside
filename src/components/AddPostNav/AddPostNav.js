import React from 'react';
import { Link } from 'react-router-dom';
import { slide as Menu } from 'react-burger-menu';
import './AddPostNav.css';
import paws from '../../images/paws-logo.png';

function AddPostNav(props) {
    return (
        <>
            <Link to='/' className='nav-title'><img className='icon' src={paws} alt='paws-up-logo'></img>{' '}<span className='title'>PawUp</span></Link>
            <>
                <Menu right>
                    <Link to='/posts' className='menu-item'>Posts</Link><br />
                    <Link to='/myPosts' className="menu-item">My Post</Link><br />
                    <Link to='/profile' className="menu-item">Profile</Link><br />
                    <Link to='/posts' className="menu-item" onClick={props.handleLogout}>Log out</Link>
                </Menu>
            </>
        </>
    )
}

export default AddPostNav;
