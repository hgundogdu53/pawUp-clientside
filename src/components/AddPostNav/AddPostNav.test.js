import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import AddPostNav from './AddPostNav';

describe(`AddPostNav component`, () => {
    it('renders without crashing', () => {
        const wrapper = shallow(<AddPostNav />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});