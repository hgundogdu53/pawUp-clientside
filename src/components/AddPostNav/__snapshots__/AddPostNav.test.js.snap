// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddPostNav component renders without crashing 1`] = `
<Fragment>
  <Link
    className="nav-title"
    to="/"
  >
    <img
      alt="paws-up-logo"
      className="icon"
      src="paws-logo.png"
    />
     
    <span
      className="title"
    >
      PawUp
    </span>
  </Link>
  <Menu
    bodyClassName=""
    burgerBarClassName=""
    burgerButtonClassName=""
    className=""
    crossButtonClassName=""
    crossClassName=""
    disableAutoFocus={false}
    disableCloseOnEsc={false}
    htmlClassName=""
    id=""
    itemClassName=""
    itemListClassName=""
    menuClassName=""
    morphShapeClassName=""
    noOverlay={false}
    noTransition={false}
    onStateChange={[Function]}
    outerContainerId=""
    overlayClassName=""
    pageWrapId=""
    right={true}
    styles={Object {}}
    width={300}
  >
    <Link
      className="menu-item"
      to="/posts"
    >
      Posts
    </Link>
    <br />
    <Link
      className="menu-item"
      to="/myPosts"
    >
      My Post
    </Link>
    <br />
    <Link
      className="menu-item"
      to="/profile"
    >
      Profile
    </Link>
    <br />
    <Link
      className="menu-item"
      to="/posts"
    >
      Log out
    </Link>
  </Menu>
</Fragment>
`;
